src/main/overview.html

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
What is this for?

Reply:
Displays on main page of javadoc. I should (and am about to) split javadoc/gradle bits into a different PR, unless you think I can just leave them in here

Reply:
They shouldn't be in here no - we're going to do a larger gradle + javadoc touchup after this is merged, so don't worry about it for now.